plugins/wireguard/access_config.go

```diff
@@ -0,0 +1,119 @@
+package wireguard
+
+import (
+	"context"
+	"fmt"
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AccessConfig() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.AccessConfig,
+		DocsURL: sdk.URL("https://www.wireguard.com/quickstart/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.PrivateKey,
+				MarkdownDescription: "Private Key of your client.",
+				Secret:              true,
+			},
+			{
+				Name:                fieldname.Address,
+				MarkdownDescription: "VPN IP address of your client.",
+				Secret:              false,
```

Comment:
nit: we can omit specifying `Secret: false`, since that's the default iirc.

Let's make sure we add mark the fields that are not required as `Optional`.